src/textual/app.py

```diff
@@ -484,6 +494,18 @@ def query(self, selector: str | None = None) -> DOMQuery:
 
         return DOMQuery(self.screen, selector)
 
+    def query_one(self, selector: str) -> DOMNode | None:
```

Comment:
I added this `query_one` method because I found it handy in the tests I was doing to chase that bug, but if you folks think that it's an unnecessary addition to the App's API I'll remove it :-)

Reply:
Yeah, not keen on this, because it classes with `DOMQuery.first`. Can discuss on Tuesday.